metric-publishers/emf-metric-publisher/src/test/java/software/amazon/awssdk/metrics/publishers/emf/EmfMetricPublisherTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.metrics.publishers.emf;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.core.metrics.CoreMetric;
+import software.amazon.awssdk.http.HttpMetric;
+import software.amazon.awssdk.metrics.MetricCategory;
+import software.amazon.awssdk.metrics.MetricCollector;
+
+import java.util.List;
+
+
+public class EmfMetricPublisherTest {
+
+    private EmfMetricPublisher.Builder publisherBuilder;
+
+    @BeforeEach
+    void setUp() {
+        publisherBuilder = EmfMetricPublisher.builder().unitTest(true);
+    }
+
+    @Test
+    void testConvertMetricCollectionToEMF_EmptyCollection(){
```

Comment:
nit:`test` prefix is redundant, see [Test naming convention](https://github.com/aws/aws-sdk-java-v2/blob/master/docs/design/NamingConventions.md#tests-naming)